code/__DEFINES/DNA.dm

```diff
@@ -114,6 +114,7 @@
 #define NOLIVER			12
 #define NOSTOMACH		13
 #define NO_DNA_COPY     14
+#define NO_REAGENTS     69
```

Comment:
Hilarious. Now make it 20 and put it where it goes

Reply:
ok bwerty